bin/frontend/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# This will build the app for production
+cmd='cd elm; npx parcel build index.html'
```

Comment:
what's the benefit of using `parcel`?

Reply:
1. We get a dev server with hot-reloading for "free" (no config needed)
2. If we ever want to extend the frontend with something like `sass` or `postcss` parcel can also handle it easily

In reality, the whole setup is a bit brittle. I'm getting an error installing `parcel` directly from nixpkgs so I was using `npx` to run it instead but it somehow works only when it wants to, unfortunately. Last Friday it worked, this Monday morning it doesn't.

I'll rethink this solution.

Reply:
Ok, I reworked how to build/serve the elm project.